db/warehouse/migrate/20250923120000_add_deleted_at_to_ce_client_proxies.rb

```diff
@@ -0,0 +1,13 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+class AddDeletedAtToCeClientProxies < ActiveRecord::Migration[7.1]
+  def change
+    add_column :ce_client_proxies, :deleted_at, :datetime
```

Comment:
note, we chose to not include deleted_at in the unique index, this makes it convenient to restore the proxy records with an upsert